commons/marmotta-sesame-tools/marmotta-rio-api/src/main/java/org/apache/marmotta/commons/sesame/rio/rdfhdt/RDFHDTFormat.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.marmotta.commons.sesame.rio.rdfhdt;
+
+import java.util.Arrays;
+
+import org.openrdf.rio.RDFFormat;
+
+/**
+ * HDT (Header, Dictionary, Triples) is a compact data structure and binary
+ * serialization format for RDF that keeps big datasets compressed to save space
+ * while maintaining search and browse operations without prior decompression.
+ * <p/>
+ * Author: Junyue Wang
+ */
+public class RDFHDTFormat {
+
+	public static final RDFFormat FORMAT = new RDFFormat("RDFHDT",
+			Arrays.asList("application/rdf+hdt"), null, Arrays.asList("hdt"),

```

Comment:
The application/rdf+hdt media type is not standardised but feel free to keep using it at this point. 

The W3C submission has the strange idea that a Media Type can be a combination of other media types, which doesn't give any top level reference to work on so this is as good as any.